### PR TITLE
Update the links on for Community and Accessibility sections

### PR DESCRIPTION
**Pull request by calvin-lau-sig7**
## What

Changes identified unclear links on the homepage, root pages and Get Started sections

## Why

This more clearly communicates what the link is and where it goes to. Part of #4267.
Note that these changes have mostly been pre-reviewed via our link audit worksheet. There's a few edits that still require edits before push to publish.

- Accessibility strategy page (links to Nova and Visual Code snippets)
- Resources and tools page

### CODE REVIEW

File: src/accessibility/accessibility-strategy/index.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ Accessibility user research often includes activities such as:
 - user interviews
 - surveys
 
-[User research must include disabled people](https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction#meeting-government-accessibility-requirements) and should also include people with a variety of access needs and impairment types.
+To ensure services are accessible, user research must include disabled people](https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction#meeting-government-accessibility-requirements) and should also include people with a variety of access needs and impairment types.
```
Comment (owenatgov) [inline src/accessibility/accessibility-strategy/index.md:268]: This is missing the opening square bracket
```suggestion
[To ensure services are accessible, user research must include disabled people](https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction#meeting-government-accessibility-requirements) and should also include people with a variety of access needs and impairment types.
```

**Review by owenatgov — COMMENTED** [review]
One little blip, otherwise looking sharp as a tack

File: src/community/resources-and-tools/index.md [hunk 2]
```diff
@@ -100,13 +100,12 @@ A Vue 3 component library for GOV.UK Frontend
 
 ## Write code
 
-You can download GOV.UK Frontend Nunjucks macro snippets for:
+[GOV.UK Design System snippets for Nova](https://extensions.panic.com/extensions/ca/ca.GOVUKDesignSystemSnippets/).
 
-- [Nova](https://extensions.panic.com/extensions/ca/ca.GOVUKDesignSystemSnippets/)
-- [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=simonwhatley.govuk-design-system-snippets)
+[GOV.UK Design System snippets for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=simonwhatley.govuk-design-system-snippets).
```
Comment (calvin-lau-sig7) [inline src/community/resources-and-tools/index.md:105]: These 2 are not approved in spreadsheet (yet) J1122 and J1123

File: src/community/develop-a-component-or-pattern/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ The Design System team focuses on developing [prioritised components and pattern
 
 If you see something you'd like to work on, join the discussion in the contribution's issue page or email the Design System team at <govuk-design-system-support@digital.cabinet-office.gov.uk>.
 
-If you have an idea for a new component or pattern that’s not already on the list, [propose it](/community/propose-a-component-or-pattern/).
+If you have an idea for a new component or pattern that’s not already on the list, [see how to propose a component or pattern](/community/propose-a-component-or-pattern/).
```
Comment (calvin-lau-sig7) [inline src/community/develop-a-component-or-pattern/index.md:14]: Not approved in spreadsheet (yet)

File: src/community/propose-a-component-or-pattern/index.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ If you'd like to work on it, read how to [develop a component or pattern](/commu
 
 ## 2. Raise an issue
 
-If your idea is not on the list, [raise an issue](https://github.com/alphagov/govuk-design-system-backlog/issues/new). Once you do, [let us know](/get-in-touch/) so a member of the Design System team can review your proposal and add it to the list.
+If your idea is not on the list, [raise an issue](https://github.com/alphagov/govuk-design-system-backlog/issues/new). Once you do, [contact us](/get-in-touch/) so a member of the Design System team can review your proposal and add it to the list.
```
Comment (calvin-lau-sig7) [inline src/community/propose-a-component-or-pattern/index.md:26]: Not approved in spreadsheet (yet)

**Comment by calvin-lau-sig7** [comment]
@owenatgov I've flagged in "files changed" the edits that are not approved in the worksheet yet. 
If you want to give your 👍 here, I'll update the spreadsheet, and we can consider this fully reviewed (with your approval).

File: src/community/propose-a-component-or-pattern/index.md (hunk 4 above)
Comment (owenatgov) [reply src/community/propose-a-component-or-pattern/index.md:26]: Both approved in worksheet 👍🏻

File: src/community/propose-a-component-or-pattern/index.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ order: 3
 
 Anyone can propose a new component or pattern for the GOV.UK Design System.
 
-Proposals need to show that the component or pattern being suggested would be [useful and unique](/community/contribution-criteria/).
+Proposals need to show that the component or pattern being suggested would [meet our contribution criteria](/community/contribution-criteria/).
```
[the remark this answers is not in the document]
Comment (owenatgov) [reply src/community/propose-a-component-or-pattern/index.md:12]: Approved in worksheet

File: src/community/resources-and-tools/index.md (hunk 2 above)
Comment (owenatgov) [reply src/community/resources-and-tools/index.md:105]: I've gone back and forth on if these should include 'Nunjucks' ie: 'GOV.UK Design System _Nunjucks_ snippets' but actually I think this is fine. If this gets flagged as unclear we'll change it.

**Review by owenatgov — COMMENTED** [review]
Final thing: squash pls

**Pull request merged**